lib/LaTeXML/Post/Manifest/Epub.pm

```diff
@@ -73,7 +73,8 @@ sub initialize {
   $package->setAttribute('version',           '3.0');
 
   # Metadata
-  my $document_metadata = $$self{db}->lookup("ID:" . $$self{db}{document_id});
+  my $rootentry         = $$self{db}->lookup('SITE_ROOT');
+  my $document_metadata = $$self{db}->lookup("ID:" . $rootentry->getValue('id'));
```

Comment:
I'm assuming that this `SITE_ROOT` mechanism achieves the same thing as your `document_id` (which doesn't get stored in the permanent db, btw).  But I didn't have any separate tests (or is there an epub test?)

Reply:
`931_epub.t` should have stopped you if you broke the epub generation badly, it at least checks that the correct files are present in the final generated `.epub` files. So likely it is ok? You can always make a DLMF epub to test :>

Reply:
Here's a cool one-liner test with a neat result:
```
latexmlc t/ams/mathtools.tex --splitat=section --split --dest=sample.epub
```

I just checked both master and your new branch and both seemed to create a successful epub with a nice ToC and working links to the specific pages. Here is the toc screenshot as a motivational demo:
![image](https://user-images.githubusercontent.com/348975/97930661-e5c27580-1d39-11eb-98cc-7d90c155e2d3.png)

So the epub side should be good :+1:

Reply:
Cool! But  now (hopefully) it should *also* work if you create a single epub from several separate documents (if you'd ever want to do that?)